doc/files/package.json.md

```diff
@@ -517,7 +517,7 @@ from the root of a package, and can be managed like any other npm
 configuration param.  See `npm-config(7)` for more on the topic.
 
 For build steps that are not platform-specific, such as compiling
-CoffeeScript or other languages to JavaScript, use the `prepublish`
+CoffeeScript or other languages to JavaScript, use the `prepare`
```

Comment:
This use case is what `prepare` is intended for: preparing the package for use. `prepublishOnly` is explicitly for the act of taking the contents of a project and putting them on the registry. The distinction matters when you use `npm install` without arguments.